### PR TITLE
add: Grid Component

### PR DESCRIPTION
**Pull request by kavyakunder**
Grids with 3 columns

grids1

Grids with 2 columns

grids2

### CODE REVIEW

File: Components/Grids/grid.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+@import "../../commonFiles/sidebar.css";
+@import "../../commonFiles/commonStyle.css";
+@import "../../commonFiles/base.css";
+
+.grid-border {
+  border: 2px solid var(--light-grey-border);
+  margin: 1rem;
+  padding: 1.5rem;
+  border-radius: 10px;
+}
```
Comment (ZuberDunge) [inline Components/Grids/grid.css:11]: suggestion : add line spacing between each {}

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: Components/Grids/grid.css (hunk 1 above)
Comment (kavyakunder) [reply Components/Grids/grid.css:11]: done!

File: Components/Grids/grid.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+.grid-content {
+  display: flex;
+  flex-direction: column;
+  font-size: 1.2rem;
+  padding-left: 3rem;
+}
+.grid-content ul li {
+  font-size: 1rem;
+  padding: 0.5rem;
```
Comment (hinanshisuthar) [inline Components/Grids/grid.css]: Use px unit for values <=10 px.

**Review by hinanshisuthar — COMMENTED** [review]
Use prettier/beautify extension to format source file. Everything else looks good to merge.

File: Components/Grids/grid.css (hunk 2 above)
Comment (kavyakunder) [reply Components/Grids/grid.css]: Done!

**Pull request merged**